features/helper_specs/helper_spec.feature

```diff
@@ -120,3 +120,96 @@ Feature: helper spec
       """
     When I run `rspec spec/helpers/widgets_helper_spec.rb`
     Then the examples should all pass
+
+  Scenario: helper methods are accessible from within example
+    Given a file named "spec/helpers/user_helper_spec.rb" with:
+      """ruby
+      require "rails_helper"
+
+      RSpec.describe UserHelper, :type => :helper do
+        describe "#email" do
+          it "gives current user email" do
+            expect(email).to eq "jon.snow@example.com"
+          end
+        end
+      end
+      """
+    And a file named "app/helpers/user_helper.rb" with:
+      """ruby
+      module UserHelper
+        def email
+          "jon.snow@example.com"
+        end
+      end
+      """
+    When I run `rspec spec/helpers/user_helper_spec.rb`
+    Then the examples should all pass
```

Comment:
I'm a bit surprised by this behavior. I think it may is a side-effect of the current implementation, but it is not the intended usage of a helper spec. I'd rather this not be documented; I actually view this as a :bug:.